@xen-orchestra/lite/src/composables/chart-theme.composable.ts

```diff
@@ -1,10 +1,19 @@
+import { useUiStore } from "@/stores/ui.store";
 import { provide } from "vue";
 import { THEME_KEY } from "vue-echarts";
 
 export const useChartTheme = () => {
+  const { colorMode } = useUiStore();
+  const isDark = colorMode === "dark";
+
+  const background = isDark ? "#14141D" : "#FFFFFF";
+  const blueScaleColor = isDark ? "#595A6F" : "#9899A5";
+  const lineColor = isDark ? "#1A1B38" : "#E5E5E7";
+  const titleColor = isDark ? "#E5E5E7" : "#1A1B38";
```

Comment:
Charts do not properly support theming. You have to reload the page to adapt the color.

Reply:
I plan to work on graph theming soon.

Since it's not directly related to this PR, this shouldn't prevent its validation.

Reply:
PR is here: https://github.com/vatesfr/xen-orchestra/pull/6587

So the current file on this PR should be reverted.

Reply:
I reverted the file